<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShelfInfoPage</class>
 <widget class="QWidget" name="ShelfInfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="shelf_name_lbl">
     <property name="text">
      <string>## ShelfName</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gLayout_1">
     <item row="0" column="0">
      <widget class="QLabel" name="info_lbl_1">
       <property name="text">
        <string>### Create on</string>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="info_lbl_2">
       <property name="text">
        <string>### Change on</string>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateTimeEdit" name="chage_datetime_edit">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="create_datetime_edit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="info_lbl_5">
     <property name="text">
      <string>### Description</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="description_te">
     <property name="documentTitle">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_2">
     <item>
      <spacer name="hSpacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="open_shelf_btn">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit_shelf_btn">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="archive_swap_btn">
       <property name="text">
        <string>Archivate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_shelf_btn">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vSpacer_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>192</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
